test/functional/test-standard-actions.js

```diff
@@ -352,6 +353,66 @@ describes.sandboxed('StandardActions', {}, () => {
       standardActions.handleAmpTarget(invocation);
       expect(printStub).to.be.calledOnce;
     });
+
+    it('should not implement print when not whitelisted', () => {
+      window.document.head.appendChild(
+          createElementWithAttributes(window.document, 'meta', {
+            name: 'amp-action-whitelist',
+            content: 'pushState,setState',
+          }));
+
+      ampdoc = new AmpDocSingle(window);
```

Comment:
Isn't this already done in `beforeEach`?

Same below for `standardActions`.

Reply:
Yes. Done